README.md

```diff
@@ -1,6 +1,7 @@
 # Octopeer
 
 ## Getting Started
+0. Change '/home/aaronang/Code' to the directory of your repository on your host machine.
```

Comment:
I would rather see this as second step. In  addition, I would use backticks to indicate the path. Please also indicate in which file this should be changed.